server/test/trigger/trigger_json.go

```diff
@@ -123,7 +123,7 @@ type triggerResultV2 struct {
 }
 
 func (tr *triggerResultV2) valid() bool {
-	return tr.HTTP != nil || tr.GRPC != nil || tr.TraceID != nil
+	return tr.HTTP != nil || tr.GRPC != nil || tr.TraceID != nil || tr.Kafka != nil
```

Comment:
This validation owes me 4 hours of debugging 🙃

Reply:
we should think of a better way of doing this, or at least making it obvious that this needs to be updated somehow